RELEASING.md

```diff
@@ -0,0 +1,138 @@
+<!-- markdownlint-configure-file { "MD013": { "line_length": 120 } } -->
+# Releasing
+
+## Overview
+
+This repo contains three types of release artifacts:
+
+* **Go Binaries** for Terraform Engine, Policy Generator, and Terraform Importer.
+* **Templates** for the Terraform Engine ("templates" for short).
+* **Policy templates** for the Policy Generator ("policies" for short).
```

Comment:
Can we link to the main README for explanation of what each of these artifacts are

Reply:
Done.